src/enums/authPages.ts

```diff
@@ -0,0 +1,7 @@
+export enum Pages {
```

Comment:
Do we need this enum? Looks like auth specific pages

Reply:
This one can be removed, yes. I added it in hopes that it might have addressed some errors coming from the common-components, but it did not, so can be removed !

Reply:
Has been removed.